src/core/reactor.cc

```diff
@@ -4185,6 +4185,10 @@ unsigned smp::adjust_max_networking_aio_io_control_blocks(unsigned network_iocbs
     seastar_logger.debug("Available AIO control blocks = aio-max-nr - aio-nr = {} - {} = {}", aio_max_nr, aio_nr, available_aio);
 
     if (available_aio < requested_aio) {
+        auto get_smp_count = [ linear_over_cpus = available_aio - std::min(reserve_iocbs, available_aio) ] (unsigned network) -> unsigned {
```

Comment:
nit, might want to remove the spaces around the capture list. to find the coding style when it comes to lambda:
```console
$ git grep -E '=\ +\[' | head -n10
apps/iotune/iotune.cc:                auto accuracy_msg = [accuracy, &rates] {
apps/memcached/ascii.rl:key = [^ ]+ >mark %{ _key = memcache::item_key(str()); };
apps/memcached/ascii.rl:        auto str = [this, &g, &p] { g.mark_end(p); return get_str(); };
apps/memcached/tests/test.py:        cmdline = [DIR_PATH + '/test_memcached.py'] + cmd
apps/memcached/tests/test_ascii_parser.cc:auto for_each_fragment_size = [] (auto&& func) {
apps/rpc_tester/rpc_tester.cc:            _call = [this] (unsigned x) { return call_echo(x); };
apps/rpc_tester/rpc_tester.cc:            _call = [this, payload = std::move(payload)] (unsigned x) { return call_write(x, payload); };
apps/rpc_tester/rpc_tester.cc:            _call = [this] (unsigned x) {
```

Reply:
Thanks! Will do so in v2.

Reply:
Done in v2.